### ISSUE TITLE
[MULTIDEVICE] Get group invite code hangs

### ISSUE THREAD

**Issue by matheust3** [pedroslopez/whatsapp-web.js#980]
### Bug description
When trying to get the invitation code of a group, the promise hangs and doesn't return anything. I just keep waiting for an answer that never arrives.

### Reproduction steps
```
const group = await message.getChat()
const inviteCode = await group.getInviteCode() // Here I wait forever
```

### Expected behavior 
I expect an invite code or some error signal

### Environment (please complete the following information):
WhatsApp

Account type [Standard / Business]: Standard
Device OS [iOS / Android]: Android
WhatsApp Web version [run await client.getWWebVersion()]: 2.2144.11
Library

Browser [Chrome / Chromium]: Chromium
NodeJS version (node -v): v14.15.1
npm or yarn version (npm -v): 6.14.8
whatsapp-web.js version: #multidevice
Other

Operating system (the one running node) [Linux / MacOS / Windows]: Linux
Operating system version (ex. Windows 10): debian:buster
Additional context
I'm using the multi device version

**Comment by KeruMx** [comment]
the message comes from a group chat?

**Comment by matheust3** [comment]
> the message comes from a group chat?

Yes, the message comes from a group of stickers.

**Comment by PurpShell** [comment]
group utilities module missing

**Comment by pedroslopez** [comment]
This has been fixed and is part of the 1.16 release. Please update and try again.
